ldms/src/store/hello_stream/Plugin_l2_stream_csv_store.man

```diff
@@ -0,0 +1,151 @@
+.fi
+.PP
+There will be a header in every output file (can be more than 1 because of rollover).
+
+.SH STORE OUTPUT FILENAME
+.PP
+The filename will be '<streamname>.<timestamp>' (e.g., foo-123456789). The timestamp is determined when the store is started or rolledover and the file is created. That may be considerably earlier than when data is streamed to the store.
+
+
+.SH STORE COLUMN ORDERING
+.PP
+There is only column ordering for 'json' format. There is no column ordering for 'str' format. 'str' format will always be written out, no matter what the 'json' header keys may be. The json order is arbitrary.
+
+.SH TIMING INFORMATION
+There are a few options for timing information. These are driven by #defines in the code source right now.
+.TP
+NDATA_TIMING
+.br
+Set by #define or #undef NDATA_TIMING and NDATA. This will write out an absolute timestamp in the log at level LINFO  every NDATA streaminput (not dictionary) lines. The format looks like "10000 lines timestamp 123456789". The timestamp is only gotten once, if the timing information is to be written out.
+.TP
+TIMESTAMP_STORE
+.br
+Set by #define or #undef TIMESTAMP_STORE. THis will write out an absolute timestamp in the file as the last item in the csv and is called 'store_recv_time' in the header. The timestamp is only gotten once, when the function is entered (e.g., if a data line has multiple dicts, this will result in multiple output lines each of which will have the same additional timestamp value.
+.PP
+
+
+.SH BUGS
+No known bugs.
+
+.SH NOTES
+.PP
+This store is in development and may be changed at any time.
+.PP
+Supports more than 1 stream. There is currently no performance guidence about number of streams and amount of data.
+.PP
+There is no way to know if a stream will actually be used or if a final value is received. Therefore, this store will need to be restarted if you want to use it with a new stream or if you want use the same stream name, but with different fields in the json.
+.PP
+It is possible that with buffering, if a stream's sends are ended, there still may be unflushed data to a file.
+.PP
+There is no way to remove a stream from the index nor to unsubscribe. That is, there is nothing that is akin to open_store and close_store pair as in an actual store plugin.
+.PP
+Note the restrictions on the data input above. Also how that affects the header.
+.PP
+This is not full featured, like the csv store. There is only one rollover, yet, for example.
+
+
+
+.SH EXAMPLES
+.PP
+Within ldmsd_controller or a configuration file:
+.nf
+load name=l2_stream_csv_store
+config name=l2_stream_csv_store path=XYZ/store container=csv stream=foo buffer=1
+# dont call anything else on the store. the store action is called by a callback triggered by the stream.
+
+prdcr_add name=localhost1 host=localhost type=active xprt=sock port=52001 interval=20000000
+prdcr_subscribe stream=foo regex=localhost*
+prdcr_start name=localhost1
+.fi
+
+.PP
+Testdata:
+.nf
+cat XXX/testdata.txt
```

Comment:
add hello_cat_publisher or other injector example here.

```
 echo '{"job-id" : 10364, "rank" : 1, "kokkos-perf-data" : [ {"name" : "SPARTAFOO0", "count": 0, "time": 0.0000},{"name" : "SPARTAFOO1", "count": 1, "time": 0.0001},{"name" : "SPARTAFOO2", "count": 2, "time": 0.0002},{"name" : "SPARTAFOO3", "count": 3, "time": 0.0003},{"name" : "SPARTAFOO4", "count": 4, "time": 0.0004},{"name" : "SPARTAFOO5", "count": 5, "time": 0.0005},{"name" : "SPARTAFOO6", "count": 6, "time": 0.0006},{"name" : "SPARTAFOO7", "count": 7, "time": 0.0007},{"name" : "SPARTAFOO8", "count": 8, "time": 0.0008},{"name" : "SPARTAFOO9", "count": 9, "time": 0.0009}] }' | hello_cat_publisher -p $port1 -a none -s foo -t json -x sock -h localhost
```

or cat the example to a file and add -f $file to invocation without pipe.

Reply:
This has been updated.